### PR TITLE
Add XML formatting to DMN/BPMN editor outputs

### LINKED ISSUE #31: pretty print the xml edited by the kogito plugin so github diff view actually works
i.e. trying to use editor editing https://github.com/quarkusio/quarkus-quickstarts/blob/master/using-kogito/src/main/resources/org/acme/kogito/persons.bpmn2 you end up with a massive diff just to change the position of a box.

The actual diff is just a single parameter change.

if the plugin would pretty print the generated xml this would be  a much smoother experience.

### PR DESCRIPTION
**Pull request by tiagobento**
This PR partially fixes #31.

There's still the issue with the ids on the BPMN editor, but diffs are now better than a one-liner.

### CODE REVIEW

File: packages/chrome-extension/package.json [hunk 1]
```diff
@@ -27,8 +27,7 @@
     "test": "echo 'No tests to run.'",
     "build:fast": "rm -rf dist && webpack",
     "build": "yarn run lint && yarn test && yarn run build:fast",
-    "build:prod": "yarn run build --mode production --devtool none",
-    "serve-envelope": "webpack-dev-server"
```
Comment (domhanak) [inline packages/chrome-extension/package.json:31]: is this line removed on purpose?

File: packages/chrome-extension/src/app/components/common/KogitoEditorIframe.tsx [hunk 2]
```diff
@@ -63,6 +63,10 @@ const RefForwardingKogitoEditorIframe: React.RefForwardingComponent<IsolatedEdit
             if (props.readonly) {
               return;
             }
+
+            //keep line breaks
+            content = content.split("\n").join("\\n");
```
Comment (domhanak) [inline packages/chrome-extension/src/app/components/common/KogitoEditorIframe.tsx:68]: just thinking out loud: Is this also windows compatible? I can give it a try later if needed.

File: packages/gwt-editors/src/DefaultXmlFormatter.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { XmlFormatter } from "./XmlFormatter";
+import * as prettifyXml from "prettify-xml";
+
+export class DefaultXmlFormatter implements XmlFormatter {
+  public format(xml: string) {
+    return prettifyXml(xml, { indent: 2, newLine: "\n" });
```
Comment (domhanak) [inline packages/gwt-editors/src/DefaultXmlFormatter.ts]: same question: Is this also windows compatible? I can give it a try later if needed.

**Review by domhanak — APPROVED** [review]
Looks good, couple of questions regarding Windows compatibility and one removed line. If the line was meant to be removed we need to update readme too.

**Comment by domhanak** [comment]
jenkins retest this

File: packages/chrome-extension/package.json (hunk 1 above)
Comment (tiagobento) [reply packages/chrome-extension/package.json:31]: Oops! Not at all :)

Comment (tiagobento) [reply packages/chrome-extension/package.json:31]: Oh, duh. Yes! Line 31 was removed on purpose. We now use this exact same command on chrome-extension-pack-kogito-kie-editors instead.

**Comment by tiagobento** [comment]
@domhanak Thx for the review! I think the `\n`s will work on Windows, but it's worth the test :)

**Review by domhanak — APPROVED** [review]

**Comment by tiagobento** [comment]
Tested on Windows. Works fine.

**Comment by tiagobento** [comment]
Tested on Windows. Works fine.

**Review by ederign — APPROVED** [review]

**Pull request merged**